File: src/my_project/easy_problems/from251to300/containing_word_with_given_prefix.py
from typing import List 

class Solution:
    def prefixCount(self, words: List[str], pref: str) -> int:
        
        count: int = 0
        
        len_pref = len(pref)
        
        for word in words:
            if word[0:len_pref] == pref:
                count += 1
        
        
        return count

File: my_project_typescript/easy_problems_typescript/from151to200/containingWordWithGivenPrefix.ts
function prefixCount(words: string[], pref: string): number {

    let count: number = 0

    let lenPref: number = pref.length

    for (let word of words){
        if (word.slice(0,lenPref) === pref){
            count += 1
        }
    }

    return count
};